index.ts

```diff
@@ -13,5 +13,24 @@ export * as symbols from './src/symbols.js'
 export { AuthManager } from './src/auth_manager.js'
 export { defineConfig } from './src/define_config.js'
 export { Authenticator } from './src/authenticator.js'
-export { withAuthFinder } from './src/mixins/with_auth_finder.js'
 export { AuthenticatorClient } from './src/authenticator_client.js'
+import type { withAuthFinder as withAuthFinderType } from './src/mixins/with_auth_finder.js'
+
+function isModuleInstalled(moduleName: string) {
+  try {
+    import.meta.resolve(moduleName)
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+// @deprecate Import `withAuthFinder` from `@adonisjs/auth/mixins` instead
+let withAuthFinder: typeof withAuthFinderType
```

Comment:
we have to use comments in this form `/*` for jsdoc. double slashes are not considered, on VSCode at least

Reply:
this is `deprecated` and not `deprecate` btw : https://jsdoc.app/tags-deprecated